### DISCUSSION TITLE
API Error

### DISCUSSION THREAD

**Discussion by drakub** [YetiForceCompany/YetiForceCRM#14416] [General]
I have been configuring to test the yetiforce API on my XAMPP installation. I successfully created the webservice application and the webservice user with a profile as portal. I am able to login and generate the toke. I am also able to fetch the fields for the different modules. But I cannot get the fields for other modules like tickets. I keep getting this error 

{
    "status": 0,
    "error": {
        "message": "No permissions for module",
        "code": 403
    }
}

Can someone help me on this

**Comment by mariuszkrzaczkowski** [comment]
The message says that there are no permissions for this module

**Reply by drakub** [reply]
But this is only with specific modules, especially tickets and FAQ. Am yet testing others. I am already doubting if the API will create records or fetch them

**Comment by drakub** [comment]
But I enabled the permissions. May be let me try the whole thing again

**Comment by drakub** [comment]
In addition to that, I have always been faced with this problem. I tried
checking on the git forum but it seems there is a patch for this error
attached here that you wrote for another person. I requested for it but
haven't got any link to it. May you please help me, Error while editing
Field.

**Comment by drakub** [comment]
Wait, I chose my web application type to be a portal, should I change it to
manage consents? What is the difference between these two?

**Reply by drakub** [reply]
I don't understand this, the error only pops for Tickets Module
